Tools/TestWebKitAPI/glib/WebKitGLib/wpe/WebViewTestWPE.cpp

```diff
@@ -37,14 +38,32 @@ void WebViewTest::resizeView(int width, int height)
     // FIXME: implement.
 }
 
-void WebViewTest::showInWindow(int, int)
+void WebViewTest::showInWindow(int width, int height)
 {
+#if ENABLE(WPE_PLATFORM)
+    if (m_display) {
+        auto* view = webkit_web_view_get_wpe_view(m_webView.get());
+        if (width && height)
```

Comment:
Shouldn't we RELEASE_ASSERT this in testing code, to make sure that doesn't happen?